APE0.rst

```diff
@@ -0,0 +1,360 @@
+* adding new Voting Members,
+* removing Voting Members,
+* electing Coordination Committee members,
+* removing Coordination Committee members,
+* appointment or removal of the Ombudsperson,
+* changes to this Charter,
+* other matters for which the Coordination Committee believes a vote is
+  appropriate.
+
+The mechanism, timeline, and criteria for a decisive vote are specified in the
+respective sections of this document in the first six cases and by the
+Coordination Committee in the last case.
+
+Membership
+^^^^^^^^^^
+Voting Members of the Astropy Project must demonstrate:
+* a good grasp of the philosophy of the Astropy Project,
+* a solid track record of being constructive and helpful,
+* significant contributions to the Project's goals, in any form,
+* a willingness to dedicate time to improving the Project,
+* a willingness to recruit, train, and nominate new team members.
+
+Voting Membership acknowledges sustained and valuable efforts that align well
+with the philosophy and the goals of the Astropy Project.
+
+Initial Membership
+^^^^^^^^^^^^^^^^^^
+Anyone who satisfies any of the following criteria at the date of acceptance of
+this APE will be invited to be in the first group of Voting Members:
+
+* has a named role in the project and has participated in an Astropy
+  coordination meeting in the last two years,
+* has a named role in the project and has regularly participated in Astropy
+  telecons (e.g., co-working telecon, spectroscopy sprint, Astropy Learn
+  telecon) in the last two years,
+* has commit rights to at least one repository in the astropy GitHub
+  organization and has actively used those commit rights in the last two years.
+
+Add New Voting Members
+^^^^^^^^^^^^^^^^^^^^^^
+Anyone can apply to become a Voting Member by providing evidence of meeting the
+requirements laid out in the Membership section above. Both self-nominations and
+nominations by others are allowed. The Voting Members are expected to make their
+decisions based on the candidate’s adherence to the membership criteria, above.
+The name of the nominee will be known to Voting Members but will not be shared
+outside Voting Members and the Coordination Committee unless the nominee becomes
+a Voting Member.
+
+The procedure for voting to add new Voting Members is:
+
+* The Coordination Committee and Ombudsperson receive each nomination, check
+  that it is factually accurate, that the nominated person accepts the
+  nomination, and that their record of community activity adheres to the Code of
+  Conduct.
+* Once a nomination is accepted by the Coordination Committee, the Coordination
+  Committee will initiate a vote among active Voting Members. The voting will be
+  open for at least two weeks and will conclude within four weeks of the
+  nomination being accepted. To successfully gain an appointment as a Voting
+  Member, the nominee must receive at least four positive votes, as long as that
+  constitutes a majority of those voting.
```

Comment:
I see why this is written with fast turn-around in mind. However, is that practical? Do we really expect anyone to organize voting once a month? Assume someone is nominated on the first of the month, voting would have to happen in that month (sending out email to all voting members, set up online voting system, look at outcome, inform community of result). That's up to 12 votes / year! Who is going to do that work? We don't turn around that fast on most of the PRs!

I suggest annually or at most twice per year, so tat you vote for a whole bunch of people at the same time (that's similar to what the IAU does). Alternatively, you could become a member if X (e.g. 4) current members second the application (similar to AAS). That means that other project members don't have the chance to vote someone down (since there won't be a voting process and they might learn about the application only after the fact), but it's much more lightweight than having a full election every time.

Reply:
I see your point here, but I think your "alternative" is pretty close to the intent of the WG.  If I'm understanding you right, the proposal in the document is to do your alternative but have a period in which people other than the "seconds" might be able to weigh in.  But they are completely free to abstain, in which case it plays out exactly like your alternative. That makes it almost as lightweight since the only added bit is setting up an online vote (which for a yes/no is really easy to do these days once you've set it up for the first time).

That said I take your point that maybe even in that model there should be a limited number per year or something. Maybe it makes sense to change the timeline to something like "4 weeks from when the nominations are announced" but say something like "announcement of nominations may be delayed by up to 6 months to ensure there are not too many elections"?  That way it's flexible enough to *allow* this without *mandating* it?

Reply:
Yes, that sounds good to me. I just want to avoid any tight deadlines that we put on ourselves. In the interest of sustainability, it's good to have the option to reduce workload by bundling a bunch of elections, but - as you say - if everything is fine, we can have them more frequently.

Reply:
See https://github.com/astropy/astropy-APEs/pull/61#issuecomment-769284716 (and the PR diff itself) for a slightly different wording which I think avoids any ambiguity